### DISCUSSION TITLE
Loading Custom dataset

### DISCUSSION THREAD

**Discussion by sahusiddharth** [keras-team/keras-cv#2011] [Q&A] [answered]
I really struggling with loading the custom data for object detection task, can you please point me some resources where i can learn, or can get a reference to get started.

The project I'm currently working on has a 3 directories and each directory has 2 directories [images, labels]. they images are of jpeg format and labels are in text file with 4 numbers xyxy separated by a space are divided into test and train

class 1
     |-----images
     |            |-------test
     |            |-------train
     |
     |-----labels
                 |-------test
                 |-------train
class 2
     |-----images
     |            |-------test
     |            |-------train
     |
     |-----labels
                 |-------test
                 |-------train
class 3
     |-----images
     |            |-------test
     |            |-------train
     |
     |-----labels
                 |-------test
                 |-------train

**Comment by jbischof** [comment]
Hi @sahusiddharth, I recommend https://keras.io/api/data_loading/

**Reply by sahusiddharth** [reply]
@jbischof Thank you,

The resource you suggested is helpful, but in case of object detection we have a bounding box as the label and that generally is stored in a single csv file or in multiple text files 

### how can i combine them for the usage in object detection ?

**Comment by jbischof** [comment, marked as answer]
For data loading questions in object detection I recommend @LukeWood 's guide: 

https://lukewood.xyz/blog/marine-animal-detection
